tabbycat/tournaments/models.py

```diff
@@ -597,3 +597,21 @@ def motions_good_for_public(self):
     @property
     def ballots_per_debate(self):
         return self.tournament.ballots_per_debate(self.stage)
+
+    @cached_property
+    def num_expected_debates(self):
+        """Returns the number of debates expected for a round with perfect team availability,
+        without checking the size of previous rounds (for elimination)."""
+        nteams = self.tournament.team_set.count()
+        if self.is_break_round:
+            rounds = list(self.tournament.round_set.filter(break_category=self.break_category).values_list('seq', flat=True).order_by('seq'))
```

Comment:
If the purpose of this query is to determine how many break rounds in the same break category precede it (I might've missed something, but it looks to me like both references are `rounds.index(self.seq)`), I think you can do this with a single query that returns the desired number 😉